alf/layers.py

```diff
@@ -3683,6 +3685,31 @@ def _make_spec(spec):
     return map_structure(_make_spec, specs)
 
 
+def to_float32(nested):
+    """Change the dtype of all the tensors in ``nested`` to torch.float32.
+
+    Args:
+        nested (nested Tensor): a nest of tensors
+    Returns:
+        nested Tensor: a nest of tensors with dtype torch.float32
```

Comment:
``Returns:
        nested Tensor``: when ``x`` is an instance of ``td.Distribution``, the return type is ``td.Distribution``?